sqlx-core/src/mysql/types/mod.rs

```diff
@@ -64,13 +64,15 @@
 //! | `uuid::Uuid`                          | BYTE(16), VARCHAR, CHAR, TEXT                        |
 //! | `uuid::fmt::Hyphenated`               | CHAR(36)                                             |
 //!
-//! ### [`json`](https://crates.io/crates/json)
+//! ### [`json`](https://crates.io/crates/serde_json)
 //!
 //! Requires the `json` Cargo feature flag.
 //!
 //! | Rust type                             | MySQL type(s)                                        |
 //! |---------------------------------------|------------------------------------------------------|
-//! | `json::JsonValue`             | JSON
+//! | [`Json<T>`]                           | JSON                                                 |
+//! | `serde_json::JsonValue`               | JSON                                                 |
+//! | `&serde_json::value::RawValue`        | JSON                                                 |
```

Comment:
Is this to update stale mysql docs, or an accidental change?

Reply:
To update stale docs